docs/01-introduction/what-is-dsp.md

```diff
@@ -3,9 +3,9 @@
  * SPDX-License-Identifier: Apache-2.0
 -->
 
-# What Is DSP and DSP-API (previous Knora)?
+# What is DSP and DSP-API (previous Knora)?
```

Comment:
does this still need Knora?

Reply:
I thought it might be helpful for users of the past. But actually you are right: I think it is obvious that the name has changed (and if not, this is not _the_ location where one would check this).